src/plugins/cordova-plugin-network-information/network.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+// Connection types from Apache Cordova network-information plugin's Connection.js implementation.
+// See https://github.com/apache/cordova-plugin-network-information/blob/master/www/Connection.js
+
+var db = require('db');
+
+var simConstants = {
+    NETWORK_TYPE_KEY: 'network-key'
+};
+
+var network = {
+    ConnectionTypes: {
+        UNKNOWN: 'unknown',
+        ETHERNET: 'ethernet',
+        WIFI: 'wifi',
+        CELL_2G: '2g',
+        CELL_3G: '3g',
+        CELL_4G: '4g',
+        CELL: 'cellular',
+        NONE: 'none'
+    },
+
+    _defaultConnectionType: 'CELL_4G',
+
+    connectionType: null,
+
+    successCallback: null,
+
+    initialize: function () {
+        this.connectionType = db.retrieve(simConstants.NETWORK_TYPE_KEY) || this._defaultConnectionType;
+    },
+
+    /**
+     * @param {string} type
+     */
+    updateNetworkType: function (type) {
+        this.connectionType = (this.ConnectionTypes[type]) ? type : 'UNKNOWN';
+
+        db.save(simConstants.NETWORK_TYPE_KEY, this.connectionType);
```

Comment:
I read again your comment, you're right, ignore my previous comment! I will make the change you are suggesting. Thanks!